index.html

```diff
@@ -80,20 +81,20 @@ <h2>services</h2>
         freegan, viral lumbersexual ennui kinfolk. Twee fixie bitters pop-up, roof party deep v 3 wolf moon irony etsy
         man braid fap jean shorts.
       </p>
-      <img src="assets/img.jpg" alt="Hello Studio Office">
+      <img src="assets/img.jpg" alt="Hello Studio Office" class="services__image">
     </section>
-    <section>
+    <section class="contact">
       <h2>contact</h2>
-      <form>
-        <label for="name">name</label>
-        <input type="text" id="name">
-        <label for="topic">topic</label>
-        <textarea id="topic"></textarea>
-        <label for="email">email</label>
-        <input type="email" id="email">
-        <button>send</button>
+      <form class="contact-form">
```

Comment:
Gdyby to był prawdziwy projekt, to pewnie by były jakieś ogólne style do formularzy i wtedy form miałby bardziej ogólną klasę no `.form` i oprócz tego mogłby mieć jakąś dodatkową, do ustalenia gdzie on się znajduje względem tej sekcji `contact` czyli np class=`form contact__form`.